### PR TITLE
EN-21594: Create local configs for SWG

### PR DESCRIPTION
**Pull request by aerust**
To kill soda2.conf!

### CODE REVIEW

File: configs/application-alpha.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+include "application.conf"
+
+com.socrata.coordinator.secondary-watcher {
+  instance = alpha
+  database.database = datacoordinator_alpha
```
Comment (znep) [inline configs/application-alpha.conf]: Is this naming consistent with other configs?

Comment (aerust) [reply configs/application-alpha.conf]: it's consistent with data-coordinator's

Comment (aerust) [reply configs/application-alpha.conf]: https://github.com/socrata-platform/data-coordinator/blob/master/configs/application-alpha.conf#L7

Comment (znep) [reply configs/application-alpha.conf]: So the idea is people have to rename their databases to switch from the old soda2.conf?

Comment (rantler) [reply configs/application-alpha.conf]: If this does indeed end up being the case, some simple tooling or at least a "copy/paste this" script would be appreciated.

Comment (aerust) [reply configs/application-alpha.conf]: No, the standard file people will use configs/sample-geocoding-secondary.conf has the current name of everyone's database. `datacoordinator`

Comment (aerust) [reply configs/application-alpha.conf]: I added essentially two configs: `application.conf` for normal people that replaces using `soda2.conf ` and the triplicate `application-alpha.conf`, etc for  people that want to run various data-coordinators for testing collocation stuff.

**Pull request merged**